LOCALIZATION.md

```diff
@@ -0,0 +1,11 @@
+# Localization
+
+Now that the first version of the cloud native glossary is live, we'd love to start translating it into different languages. Here are the basic requirements for a localization team:
+
+* At least three volunteers
+* Two approvers Each term must be approved by two approvers
+* Unanimous consent must be reached to be approved 
+
+## Instructions
+
+Translations of glossary terms should go in the appropriate language directory inside of `/content/`.  New languages need to be set up inside of `/config.toml`.
```

Comment:
How about adding following contents ?

- Minimum required content or page as the first l10n contribution
  - ex: https://github.com/cncf/glossary/blob/main/content/en/_index.md
  
- Adding glossary/i18n/`{lang}`.toml (do we need? :) )

- Example of `New languages need to be set up inside of /config.toml` 
  - like .. https://github.com/cncf/glossary/blob/46cfc9bf283612bbc55a7b979c82454b15b47e1d/config.toml#L54
     ```diff
     [languages]
    [languages.en]
    title = "Cloud Native Glossary"
    description = "The CNCF Cloud Native Glossary Project is intended to be used as a reference for common terms used when talking about cloud native applications."
    languageName ="English"
    # Weight used for sorting.
    weight = 1
    
    +[languages.ko]
    +title = "클라우드 네이티브(Cloud Native) 용어집"
    +description = "CNCF 클라우드 네이티브 용어집 프로젝트는 클라우드 네이티브 애플리케이션에 대한 대화를 나눌 때 공통의 용어를 참조하여 사용하도록 하는 목적을 가지고 있다."
    +languageName ="Korean"
    +weight = 2
    ```

Reply:
@seokho-son, that's a great idea. What if I close this PR and you create the localization doc?

Reply:
@CathPag I opened #257 :)